components/Breadcrumb/src/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import BaseProps from '@gemeente-denhaag/baseprops';
+import './breadcrumb.css';
+
+export interface BreadcrumbProps extends BaseProps {}
+
```

Comment:
I know this PR isn't focussed on React, but to avoid confusion when the React component gets implemented. I think it's better if we can remove any usages of BaseProps. Also, the BreadcrumbProps interface is unused for now, so we can also remove that.

Reply:
Done